Source/DafnyCore/Dafny.atg

```diff
@@ -161,26 +165,41 @@ public static int Parse (string/*!*/ s, string/*!*/ fullFilename, string/*!*/ fi
 }
 
 public static Parser SetupParser(string/*!*/ s, string/*!*/ fullFilename, string/*!*/ filename, Include include, ModuleDecl module,
-                                 BuiltIns builtIns, Errors/*!*/ errors, bool verifyThisFile=true, bool compileThisFile=true) {
+                                 BuiltIns builtIns, Errors/*!*/ errors, bool verifyThisFile=true, bool compileThisFile=true,
```

Comment:
Is there a test that can cover these changes to the grammar? If not, can you please add one?

Reply:
Either way are fine for me. I know adding a new test is cumbersome so I understand if you want to remove this extra code. I'm happy to review the PR from AutoExterns that will also contain the tests.